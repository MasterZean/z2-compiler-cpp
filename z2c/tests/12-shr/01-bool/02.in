false >> true
false >> false
false >> 4s8
false >> 4u8
false >> 4s16
false >> 4u16
false >> 4
false >> 4u
false >> 4l
false >> 4ul
false >> 4.0f
false >> 4.0
false >> '4'
false >> 4p
