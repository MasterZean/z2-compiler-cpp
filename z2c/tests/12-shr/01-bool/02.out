DummyClass(1, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Bool'
DummyClass(2, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Bool'
DummyClass(3, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Small'
DummyClass(4, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Byte'
DummyClass(5, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Short'
DummyClass(6, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Word'
DummyClass(7, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Int'
DummyClass(8, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'DWord'
DummyClass(9, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Long'
DummyClass(10, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'QWord'
DummyClass(11, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Float'
DummyClass(12, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Double'
DummyClass(13, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Char'
DummyClass(14, 7): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'PtrSize'
